src/benchmark_mediation.py

```diff
@@ -617,174 +619,309 @@ def alternative_estimator(y, t, m, x, regularization=True):
+    Presented in Eric J. Tchetgen Tchetgen. Ilya Shpitser.
     "Semiparametric theory for causal mediation analysis: Efficiency bounds,
     multiple robustness and sensitivity analysis."
     Ann. Statist. 40 (3) 1816 - 1845, June 2012.
     https://doi.org/10.1214/12-AOS990
 
-    m is binary !!!
+    Parameters
+    ----------
+    y : array-like, shape (n_samples)
+        Outcome value for each unit, continuous
 
-    y       array-like, shape (n_samples)
-            outcome value for each unit, continuous
+    t : array-like, shape (n_samples)
+        Treatment value for each unit, binary
 
-    t       array-like, shape (n_samples)
-            treatment value for each unit, binary
+    m : array-like, shape (n_samples)
+        Mediator value for each unit, necessarly binary and unidimensional
 
-    m       array-like, shape (n_samples)
-            mediator value for each unit, here m is necessary binary and uni-
-            dimensional
+    x : array-like, shape (n_samples, n_features_covariates)
+        Covariates value for each unit, continuous
 
-    x       array-like, shape (n_samples, n_features_covariates)
-            covariates (potential confounders) values
+    interaction : boolean, default=False
+        Whether to include interaction terms in the model
+        interactions are terms XT, TM, MX
 
-    interaction boolean, default=False
-                whether to include interaction terms in the model
-                interactions are terms XT, TM, MX
+    forest : boolean, default=False
+        Whether to use a random forest model to estimate the propensity
+        scores instead of logistic regression, and outcome model instead
+        of linear regression
 
-    penalty string {‘l1’, ‘l2’, ‘elasticnet’, ‘none’}, default=’l2’
-            passed to the scikit learn sklearn.linear_model.LogisticRegression
-            model. Was used to mimick the R implementation without penalty in
-            some cases, to evaluate regularization bias
+    crossfit : integer, default+0
+        Number of folds for cross-fitting. If crossfit<2, no cross-fitting is applied
 
-    forest  boolean, default False
-            whether to use a random forest model to estimate the propensity
-            scores instead of logistic regression, and outcome model instead
-            of linear regression
+    clip : float, default=0.01
+        Limit to clip p_x and f_mtx for numerical stability (min=clip, max=1-clip)
 
-    crossfit integer, default 0
-             number of folds for cross-fitting
+    regularization : boolean, default=True
+        Whether to use regularized models (logistic or linear regression).
+        If True, cross-validation is used to chose among 8 potential
+        log-spaced values between 1e-5 and 1e5
 
-    clip    float
-            limit to clip for numerical stability (min=clip, max=1-clip)
+    calibration : boolean, default=True
+        [...]
```

Comment:
to complete, suggestion 
```
Whether to add a calibration step so that the classifier used to estimate the treatment propensity score and the density of the (binary) mediator. Calibration ensures the output of the [predict_proba](https://scikit-learn.org/stable/glossary.html#term-predict_proba) method can be directly interpreted as a confidence level.
```

Reply:
done, [017e5b9](https://github.com/judithabk6/med_bench/pull/35/commits/017e5b98b26ce4f33c14311e53152246d411a53d)